app/views/volunteers/_form.html.slim

```diff
@@ -1,13 +1,12 @@
 .row
   .col-xs-12
     = f.error_notification
-
 .row
-  .col-xs-12.col-md-6
-    = f.input :first_name
-
-  .col-xs-12.col-md-6
-    = f.input :last_name
+  .col-xs-12
```

Comment:
no need to nest the contact fields in `.row .col-xs-12` as they have their own bootstrap grid structure allready.